llvm/test/tools/llvm-cov/mcdc-maxbs.test

```diff
@@ -0,0 +1,21 @@
+# Test the assertion failure in llvm-cov with empty bitmap (#78922)
+# REQUIRES: asserts
+
+# RUN: llvm-profdata merge %S/Inputs/mcdc-maxbs.proftext -o %t.profdata
+# RUN: llvm-cov report --show-mcdc-summary %S/Inputs/mcdc-maxbs.o -instr-profile %t.profdata
+
+# Instructions for regenerating the test object:
+
+clang -O3 -fcoverage-mcdc -fprofile-instr-generate \
+    -fcoverage-mapping -fcoverage-compilation-dir=. \
+    mcdc-maxbs.c -c -o mcdc-maxbs.o
```

Comment:
You may want to note `cd $S/Inputs `to underscore the location of mcdc-maxbs.c

Reply:
These are assumed to be done in `$S/Inputs`. I think it is not a bad manner since they are done manually.
I will add `cd $S/Inputs # Or copy %S/Inputs/mcdc-maxbs.c to the working directory`.

Reply:
Thanks. Also, I should've said `%S/Inputs` rather than `$S/Inputs`. sorry for the typo.